axelor-account/src/main/resources/reports/Invoice.rptdesign

```diff
@@ -6229,7 +6229,7 @@ else {
                                     <text id="5768">
                                         <property name="marginTop">5pt</property>
                                         <property name="contentType">html</property>
-                                        <text-property name="content"><![CDATA[<VALUE-OF format="MM/dd/yyyy">if(params["Locale"].value == 'en')row["StockMoveRealDate"]</VALUE-OF><VALUE-OF format="dd/MM/yyyy">if(params["Locale"].value != 'en')row["StockMoveRealDate"]</VALUE-OF> - <VALUE-OF>row["StockMoveSeq"]</VALUE-OF>
+                                        <text-property name="content"><![CDATA[<VALUE-OF format="MM/dd/yyyy">if(params["Locale"].value == 'en_US')row["StockMoveRealDate"]</VALUE-OF><VALUE-OF format="dd/MM/yyyy">if(params["Locale"].value != 'en_US')row["StockMoveRealDate"]</VALUE-OF> - <VALUE-OF>row["StockMoveSeq"]</VALUE-OF>
```

Comment:
this is a shitty code, you can't do anything about it for now but we'll have to improve it later, it should not be like this, it should be like any other date in the report